ocs_ci/ocs/machine.py

```diff
@@ -275,6 +298,47 @@ def create_custom_machineset(
         raise UnsupportedPlatformError("Functionality not supported in UPI")
 
 
+def create_infra_nodes(num_nodes):
```

Comment:
create_custom_machineset takes m4.xlarge as default input. Are we creating infra worker nodes with "m5.4xlarge" ?

Reply:
This should depend on config value, or it can be parametrized in the funciton with None as default value, and if passed by function parameter it will use this value, otherwise it will use value from config and if not defined in config it can use some default value - e.g. same as worker node.

Reply:
I agree on this, This can be parametrized according to the user